src/prpy/util.py

```diff
@@ -749,3 +749,86 @@ def IsTimedTrajectory(trajectory):
     """
     cspec = trajectory.GetConfigurationSpecification()
     return cspec.ExtractDeltaTime(trajectory.GetWaypoint(0)) is not None
+
+
+def jointposvelacc_from_traj(robot, traj, time):
+    """
+    Helper function to extract the joint position, velocity and acceleration
+    from an OpenRAVE trajectory.
+    @param robot The OpenRAVE robot
+    @param traj An OpenRAVE trajectory
+    @param time List of times in seconds
+    @return pva_list List of list of position, velocity and acceleration
+                     at specified times. Inserts 'None' for unavailable fields
+    """
+
+    duration = traj.GetDuration()
+
+    if any(time > duration):
+        raise ValueError('Input time {0:} exceeds duration {1:.2f}'
+                         .format(time, duration))
+
+    cspec = traj.GetConfigurationSpecification()
+    num_dofs = robot.GetDOF()
+    dof_indices = range(num_dofs)
+
+    pva_list = []
+    for t in time:
+        pva = []
```

Comment:
I didn't do this because some `pva` entries can be `None` and some weird behavior ensues:
```
In [101]: foo = numpy.zeros((6,3))

In [102]: foo[1] = None

In [103]: foo
Out[103]: 
array([[  0.,   0.,   0.],
       [ nan,  nan,  nan],
       [  0.,   0.,   0.],
       [  0.,   0.,   0.],
       [  0.,   0.,   0.],
       [  0.,   0.,   0.]])

In [104]: foo[1]
Out[104]: array([ nan,  nan,  nan])

In [105]: foo[1] is None
Out[105]: False
```